sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/ReadFeedDatabasesTest.java

```diff
@@ -36,11 +36,9 @@ public ReadFeedDatabasesTest(CosmosClientBuilder clientBuilder) {
 
     @Test(groups = { "simple" }, timeOut = FEED_TIMEOUT)
     public void readDatabases() throws Exception {
-
-        FeedOptions options = new FeedOptions();
         int maxItemCount = 2;
 
-        CosmosPagedFlux<CosmosDatabaseProperties> feedObservable = client.readAllDatabases(options);
```

Comment:
the page size validation may fail.

```java
        int expectedPageSize = (allDatabases.size() + maxItemCount - 1) / maxItemCount;
        FeedResponseListValidator<CosmosDatabaseProperties> validator = new FeedResponseListValidator.Builder<CosmosDatabaseProperties>()
                .totalSize(allDatabases.size())
                .exactlyContainsInAnyOrder(allDatabases.stream().map(d -> d.getResourceId()).collect(Collectors.toList()))
                .numberOfPages(expectedPageSize)
                .pageSatisfy(0, new FeedResponseValidator.Builder<CosmosDatabaseProperties>()
                        .requestChargeGreaterThanOrEqualTo(1.0).build())
                .build();

        validateQuerySuccess(feedObservable.byPage(maxItemCount), validator, FEED_TIMEOUT);
```

Reply:
I'm not seeing any test failures when run locally...
I'm not sure I understand the concern since we already moved the maxItemCount out of the FeedOptions when we transitioned to return PageFlux results... Plus under the new API we do exactly what the test does, which is to call the now package private readAll*() API with a "new FeedOptions()" argument value.